src/java/fr/paris/lutece/portal/service/daemon/AppDaemonService.java

```diff
@@ -295,8 +295,9 @@ private static void scheduleThread( DaemonEntry entry )
      */
     private static void scheduleThread( DaemonEntry entry, int nInitialDelay )
     {
-        _executor.schedule( entry, nInitialDelay, TimeUnit.SECONDS );
+        AppLogService.info( "Scheduling daemon " + entry.getId( ) + "; first run in " + nInitialDelay + " seconds" );
         entry.setIsRunning( true );
+        _executor.schedule( entry, nInitialDelay, TimeUnit.SECONDS );
         // update onStartup property
```

Comment:
Ah I see, I misunderstood the commit message. ( I thought this commit was mainly about adding logs, ie "indicate in the log", but you meant "indicate" as "setIsRunning(true);" and the commit is mainly about fixing the bug ! )
Great, thanks !